cached_network_image/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: cached_network_image
 description: Flutter library to load and cache network images.
   Can also be used with placeholder and error widgets.
 homepage: https://github.com/Baseflow/flutter_cached_network_image
-version: 3.2.0
+version: 3.2.1
```

Comment:
Are we sure that it's just a minor change? Maybe we should bump it by 0.1.0?

Reply:
I wasn't sure either, but since the API of this package isn't changing I think this is the most correct 🤔